test/ServerSpec.js

```diff
@@ -65,4 +67,115 @@ describe('Server Spec', () => {
         .end(done);
     });
   });
+
+  describe('/search/:market/:checkin/:checkout', () => {
+    it('Should retrieve all San Francisco listings available for checkin 2017-11-12 checkout 2017-11-13', (done) => {
+      request(server)
+        .get('/search/San%20Francisco/2017-11-12/2017-11-13')
+        .expect('Content-Type', /json/)
+        .expect(200)
+        .expect((response) => {
+          const availableListings = response.body.filter(result => result.inventory_date.split('T')[0] === '2017-11-12');
+          expect(availableListings.length).to.equal(response.body.length);
```

Comment:
I know we discussed this but just to confirm, this is just checking that every return entry is equal to the check-in date. It doesn't actually guarantee all SF listings for those dates are being retrieved.

Reply:
That is correct. I can actually also check for length = 4 here since that's how many there are in my test db that match this search criteria.